core/modules/dashboard/dashboard.module

```diff
@@ -0,0 +1,444 @@
+    'class' => 'DashboardOverviewUserBlock',
+  );
+  $blocks['updates'] = array(
+    'info' => t('Dashboard - Available updates'),
+    'description' => t('Displays a list of available updates for modules, themes, layouts, and Backdrop core.'),
+    'class' => 'DashboardUpdateBlock',
+  );
+
+  if (module_exists('taxonomy')) {
+    $blocks['taxonomy'] = array(
+      'info' => t('Dashboard - Categories'),
+      'description' => t('Provides links to manage taxonomy'),
+      'class' => 'DashboardTaxonomyBlock',
+    );
+  }
+
+  return $blocks;
+}
+
+/**
+ * Implements hook_autoload_info().
+ */
+function dashboard_autoload_info() {
+  return array(
+    'DashboardCreateBlock' => 'includes/block.create.inc',
+    'DashboardMenuBlock' => 'includes/block.menus.inc',
+    'DashboardContentTypesBlock' => 'includes/block.node_types.inc',
+    'DashboardOverviewContentBlock' => 'includes/block.overview_content.inc',
+    'DashboardOverviewUserBlock' => 'includes/block.overview_user.inc',
+    'DashboardTaxonomyBlock' => 'includes/block.taxonomy.inc',
+    'DashboardUpdateBlock' => 'includes/block.update.inc',
+  );
+}
+
+/**
+ * Implements hook_user_presave().
+ */
+function dashboard_user_presave($account) {
+  cache_clear_all('dashboard_user_overview', 'cache');
+}
+
+/**
+ * Implements hook_user_delete().
+ */
+function dashboard_user_delete($account) {
+  cache_clear_all('dashboard_user_overview', 'cache');
+}
+
+/**
+ * Implements hook_user_login().
+ */
+function dashboard_user_login(&$edit, $account) {
+  // Don't redirect if a destination is already set.
+  if (!empty($_GET['destination'])) {
+    return;
+  }
+
+  // Don't redirect for password reset forms.
+  if (isset($_POST['form_id']) && $_POST['form_id'] == 'user_pass_reset') {
+    return;
+  }
+
+  // Don't redirect if no permission to access dashboard.
+  if (!user_access('access dashboard', $account)) {
+    return;
+  }
+
+  // Don't redirect if redirect setting is not set.
+  if (config_get('dashboard.settings', 'dashboard_login_redirect') != 1) {
+    return;
+  }
+
+  // Check access to layout and redirect.
+  $dashboard = layout_load('dashboard');
+  if ($dashboard && !$dashboard->disabled && $dashboard->checkAccess()) {
+    $_GET['destination'] = 'admin/dashboard';
+  }
+}
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function dashboard_preprocess_page(&$variables) {
+  if (arg(0) == 'admin' && arg(1) == 'dashboard') {
+    // Include dashboard specific style sheet.
+    $css_path = backdrop_get_path('module', 'dashboard') . '/css/dashboard.css';
+    backdrop_add_css($css_path);
+    $variables['classes'][] = 'dashboard';
+  }
+}
+
+/**
+ * Implements hook_preprocess_block().
+ */
+function dashboard_preprocess_block(&$variables) {
+  if ($variables['layout']->name == 'dashboard'
+     && $variables['block']->module == 'dashboard') {
+    // Add dashboard specific block class.
+    $variables['classes'][] = 'admin-panel';
```

Comment:
A code comment indicating `admin-panel` is already a special/generic Backdrop class rather than being dashboard-specific might be helpful. Otherwise this should have been namespaced under `dashboard-`.

Reply:
This was fixed in @quicksketch 's PR.